src/shared/components/BaseToastMessages.tsx

```diff
@@ -0,0 +1,153 @@
+import {
+  Card,
+  Link,
+  CancelIcon,
+  CautionIcon,
+  CheckIcon,
+} from '@hypothesis/frontend-shared/lib/next';
+import classnames from 'classnames';
+
+import type { ToastMessage } from '../../sidebar/store/modules/toast-messages';
```

Comment:
In order to preserve the dependency structure, we'll need to move this type into a module in `src/shared/`, which `sidebar/store/modules/toast-messages` can then reference. The type could live in this module for example.

Reply:
True. I forgot I wanted to discuss this topic.